## Relation: Throw

Descriptions: An `Throw Exception Relation` is an error condition, possibly outside the program's control, that prevents the program from continuing along its regular execution path. Certain operations, including object creation, file input/output, and function calls made from other modules, are all potential sources of exceptions, even when your program is running correctly.

### Supported Patterns

```yaml
name: Throw Exception Declaration
```
#### Syntax: Catch Exception

```text
noexcept noexcept(true) throw()
```

##### Examples

###### Try Catch Exception


###### Throw Exception
